### PR TITLE
Fix prereg JS errors

### PR DESCRIPTION
**Pull request by kitsuta**
Fixes https://github.com/magfest/ubersystem/issues/1626, which was a bug exposed by a configuration change this year. Also fixes a previously-unknown bug with groups.

### CODE REVIEW

**Comment by binary1230** [comment]
+111111111111111111111111111111

**Comment by binary1230** [comment]
followup: #1626 only showed up when supporter deadline was different than printed badge deadline.  

FUNNY STORY: for m13, both of these deadlines were identical so even though this bug existed, we didn't see it.  

for m14, the values are different (Jan 10th vs 20th), and the gap was causing the hidden elements to not display, causing the calculation of $.val('amount_extra') to crash out and tank the page.

Tag teaming this shit with @kitsuta, good stuff.

It occurs to me we might want to just do a quick search of other places where this might have been an issue.  but this will definitely fix the big major issues with the prereg confirmation page for supporters and higher.

File: uber/templates/preregistration/preregbase.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
                 {% if c.BEFORE_GROUP_PREREG_TAKEDOWN %}
                     description: '<p class="list-group-item-text">Register a group of {{ c.MIN_GROUP_SIZE }} people or more.</p>',
                 {% else %}
-                    description: '<p class="list-group-item-text">The deadline for Group registration has passed, but you can still register as a regular attendee.</p>'
+                    description: '<p class="list-group-item-text">The deadline for Group registration has passed, but you can still register as a regular attendee.</p>',
```
Comment (EliAndrewC) [inline uber/templates/preregistration/preregbase.html]: Ha, whoops!  Good catch!

**Comment by EliAndrewC** [comment]
+1

**Pull request merged**